apps/playground/src/app/modules/components/search-bar-module/search-bar-module.component.html

```diff
@@ -9,168 +9,196 @@ <h2>DsfrSearchBarComponent</h2>
     It can be used on its own :
   </p>
   <ul>
-      <li>as a simple input, to retrieve the search query entered by the user after they click on the search button, hook onto the <code>searchQuerySubmitted</code> Output property</li>
-      <li>as an input with autocompletion, to retrieve the search result chosen by the user, hook onto the <code>searchResultSelected</code> Output property</li>
+    <li>as a simple input, to retrieve the search query entered by the user after they click on the search button, hook
+      onto the <code>searchQuerySubmitted</code> Output property
+    </li>
+    <li>as an input with autocompletion, to retrieve the search result chosen by the user, hook onto the <code>searchResultSelected</code>
+      Output property
+    </li>
   </ul>
   <p>
     It can also be used as a <code>FormControl</code>, with the ReactiveForms API.
   </p>
   <p>
     It is based on the
     <a
-      target="_blank"
-      hreflang="fr"
-      title="The documentation page for the Search Bar Component in the DSFR"
-      href="https://www.systeme-de-design.gouv.fr/elements-d-interface/composants/barre-de-recherche/">
+      target='_blank'
+      hreflang='fr'
+      title='The documentation page for the Search Bar Component in the DSFR'
+      href='https://www.systeme-de-design.gouv.fr/elements-d-interface/composants/barre-de-recherche/'>
       Search Bar
       <mat-icon>open_in_new</mat-icon>
     </a>
     component in the
     <a
-      target="_blank"
-      hreflang="fr"
-      title="The DSFR documentation"
-      href="https://www.systeme-de-design.gouv.fr/">
+      target='_blank'
+      hreflang='fr'
+      title='The DSFR documentation'
+      href='https://www.systeme-de-design.gouv.fr/'>
       DSFR
       <mat-icon>open_in_new</mat-icon>
     </a>
     .
   </p>
-  <p>It has several <code>@Input()</code>  and <code>@Output()</code> properties:</p>
-  <dl class="input-properties">
+  <p>It has several <code>@Input()</code> and <code>@Output()</code> properties:</p>
+  <dl class='input-properties'>
     <play-input-property-definition
-      name="id"
-      [required]="true"
+      name='id'
+      [required]='true'
       type="It takes a string with unique value'">
     </play-input-property-definition>
     <play-input-property-definition
-      name="label"
-      [required]="true"
-      type="It takes a string.">
+      name='label'
+      [required]='true'
+      type='It takes a string.'>
       The label of the input search, only for accessibility because it is not displayed
```

Comment:
`The label of the input search, necessary for accessibility. It won't be displayed unless <code>hasSearchButton</code> is <code>false</code>`